### PR TITLE
[14.0][IMP] shopinvader menues and config.settings

### PR DESCRIPTION
**Pull request by ivantodorovich**
- Moved `shopinvader.config.settings` to a real `res.config.settings` page
- Reorganized menues in submenues
- Added menu for `shopinvader.product`

**Before:**
https://user-images.githubusercontent.com/1914185/128345875-f9c95a68-8005-43c5-94ee-a7a4ad87d4c7.mp4

**After:**
https://user-images.githubusercontent.com/1914185/128345898-a5ef4c79-e270-4a56-848d-4644c935d939.mp4

### CODE REVIEW

File: shopinvader/models/res_partner.py [hunk 1]
```diff
@@ -48,6 +48,18 @@ class ResPartner(models.Model):
         store=True,
     )
 
+    @api.model
+    def _is_partner_duplicate_allowed(self):
+        """Check if partner duplication is allowed
```
Comment (simahawk) [inline shopinvader/models/res_partner.py:55]: https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings

File: shopinvader/models/res_partner.py [hunk 2]
```diff
@@ -48,6 +48,18 @@ class ResPartner(models.Model):
...
+        This parameter is configured through res.config.settings
+        """
+        get_param = self.env["ir.config_parameter"].sudo().get_param
+        param = get_param("shopinvader.no_partner_duplicate")
+        if not param:
+            return True  # default value
+        else:
+            return param == "False"
```
Comment (simahawk) [inline shopinvader/models/res_partner.py]:
```py
from odoo.tools.misc import str2bool
[...]
return str2bool(param)
```

```py
>>> str2bool(None, default=True)
True
```

**Review by simahawk — APPROVED** [review]

File: shopinvader/views/shopinvader_menu.xml [hunk 3]
```diff
@@ -18,4 +29,32 @@
...
+    <menuitem
+        id="menu_shopinvader_config_products"
+        name="Products"
+        parent="menu_shopinvader_config"
+        sequence="10"
+    />
+
+    <menuitem
+        id="menu_shopinvader_config_partners"
+        name="Products"
```
Comment (simahawk) [inline shopinvader/views/shopinvader_menu.xml]:
```suggestion
        name="Partners"
```

**Review by simahawk — CHANGES_REQUESTED** [review]
can you post a screenshot of the result?

**Review by rousseldenis — APPROVED** [review]

**Comment by rousseldenis** [comment]
@ivantodorovich Could you attend @simahawk comments ?

**Comment by ivantodorovich** [comment]
Ready now, applied @simahawk comments and added before / after screenshots on the description :)

**Review by simahawk — APPROVED** [review]

**Pull request merged**